flink-connector-mongodb-cdc/src/main/java/com/ververica/cdc/connectors/mongodb/table/MongoDBConnectorFullChangelogDeserializationSchema.java

```diff
@@ -37,6 +39,8 @@
  * Deserialization schema from Mongodb ChangeStreamDocument to Flink Table/SQL internal data that
  * produces full changelog mode structure {@link RowData}.
  */
+@Experimental
+@PublicEvolving
 public class MongoDBConnectorFullChangelogDeserializationSchema
```

Comment:
minor: We can only choose one of the two annotation, how about remove `@Experimental`?